store-conf/conf/log4j.properties.production

```diff
@@ -207,8 +207,8 @@ logger.memcached.name = net.spy.memcached
 logger.memcached.level = WARN
 
 %%comment VAR:!zimbraLogHostname%%logger.slogger.name = zimbra.slogger
-%%comment VAR:!zimbraLogHostname%%logger.slogger.level=INFO
-%%uncomment VAR:!zimbraLogHostname%%logger.slogger.level=ERROR
+%%comment VAR:!zimbraLogHostname%%logger.slogger.level = info
+%%uncomment VAR:!zimbraLogHostname%%logger.slogger.level = error
```

Comment:
Can you please actually test this with `zimbraLogHostname` unset ?  Now it results in:
```
#logger.slogger.name = zimbra.slogger
#logger.slogger.level = info
logger.slogger.level = error
#logger.slogger.additivity = false
#logger.slogger.appenderRef.SLOGGER.ref = sloggerAppender
```
So again no logs (like before) because slogger is undefined:

`ERROR: service.FAILURE (system failure: org.apache.logging.log4j.core.config.ConfigurationException: No name attribute provided for Logger slogger)`